docs/NEW_GUIDE_DRAFT.en.md

```diff
@@ -0,0 +1,81 @@
+`NEAR CLI` is using `extensions` to saticfy all groups of users.
+    - `Core NEAR CLI` commands should be usefull for all groups of users.
+    - `Core NEAR CLI` is a single binary, there is no extensions that are installed `by default`.
+    - Each extension is a separate binary that can be executed from `NEAR CLI`.
+    - Extensions are used by a particular group or several groups of users.
+    - Extensions are not composable (you should not create extensions for extensions).
+
+`NEAR CLI` is built for:
+    - NEAR `dApp developers`, who build smart-contracts, UIs, and tooling on NEAR.
+    - `Tech-savvy` people automating their routines.
+    - `Validators`
+    - ...
+
+`Core NEAR CLI` commands:
+import-account
+    - from-near-wallet
+    - from-seed-phrase
+    - from-ledger
+    - from-private-key
+create-account
+    - implicit
+    - subaccount
+manage-account `<- accountId is entered in this step`
+    - get-state
+    - keys
+        - view
+        - add
+        - delete
+    - contract
+        - deploy-code
+        - dev-deploy-code
+        - get-code-checksum
+        - get-state
+        - call-view-method
+        - call-change-method
+    - delete
+transfer
+helpers
+    - generate-key
```

Comment:
We can have `off-chain-keys` top-level command that can have these two helpers + local keys management.
Also, we can rename `manage-account` -> `keys` to `manage-account` -> `on-chain-keys`.

Reply:
Helpers indeed contradict the extensions concept, so I agree with the plan to put these under some top-level command, but I don't have any good ideas at the moment. I don't like the asymmetry of `manage-account -> on-chain-keys` vs just `off-chain-keys`.